src/context/Theme.tsx

```diff
@@ -56,8 +56,8 @@ const tonalOffset = 0.1;
 const xl = 1800;
 const lg = 1200;
 const md = 900;
-const sm = 600;
-const xs = 500;
+const sm = 650;
+const xs = 0;
```

Comment:
This is SUCH a weird thing. We need to set `xs` to zero so that JSON Forms does not disappear the forms when you shrink the view really narrow. This combine with down below where was set the minwidth to `sm` it is fine having `xs` at 0.